includes/actions.php

```diff
@@ -245,3 +245,19 @@ function give_set_donation_levels_max_min_amount( $form_id ) {
 }
 
 add_action( 'give_pre_process_give_forms_meta', 'give_set_donation_levels_max_min_amount', 30 );
+
+/**
+ * This function will clear seo sitemap cache on update of settings
+ *
+ * @since 1.8.9
+ *
+ * @return void
+ */
+function give_clear_seo_sitemap_cache_on_settings_change() {
+	// If Yoast SEO plugin exists, then update seo sitemap cache.
+	if( is_plugin_active( 'wordpress-seo/wp-seo.php' ) ) {
+		WPSEO_Sitemaps_Cache::register_clear_on_option_update( 'give_settings', '' );
```

Comment:
@ravinderk As we have all the data of admin settings saved under a single option, I've used it `WPSEO_Sitemaps_Cache::register_clear_on_option_update( 'give_settings', '' );`. But, I'll check if there is any way out to manage this in a way you suggested and let you know.